src/plugins/saved_objects_management/public/management_section/recent_work.tsx

```diff
@@ -258,56 +258,65 @@ export const RecentWork = (props: { core: CoreStart; workspaceEnabled?: boolean
-                                  {
-                                    defaultMessage: 'Viewed',
-                                  }
-                                )
-                              : i18n.translate(
-                                  'savedObjectsManagement.recentWorkSection.updatedAt',
-                                  {
-                                    defaultMessage: 'Updated',
-                                  }
-                                )}
-                            :{' '}
-                          </EuiText>
-                        </EuiFlexItem>
-                        <EuiFlexItem grow={1} style={{ textAlign: 'right' }}>
-                          <EuiText size="xs" color="default">
-                            <b>
-                              {selectedSort === recentlyViewed
-                                ? moment(recentAccessItem?.lastAccessedTime).fromNow()
-                                : moment(recentAccessItem?.updatedAt).fromNow()}
-                            </b>
-                          </EuiText>
-                        </EuiFlexItem>
-
-                        {workspaceEnabled && (
-                          <>
+                      <EuiFlexGroup
+                        justifyContent="spaceBetween"
+                        direction="column"
+                        gutterSize="none"
+                      >
+                        <EuiFlexItem>
+                          <EuiFlexGrid columns={2} gutterSize="s">
                             <EuiFlexItem grow={false}>
                               <EuiText size="xs" color="default">
-                                {i18n.translate(
-                                  'savedObjectsManagement.recentWorkSection.workspace',
-                                  {
-                                    defaultMessage: 'Workspace',
-                                  }
-                                )}
-                                :
+                                {selectedSort === recentlyViewed
+                                  ? i18n.translate(
+                                      'savedObjectsManagement.recentWorkSection.viewedAt',
+                                      {
+                                        defaultMessage: 'Viewed',
+                                      }
+                                    )
+                                  : i18n.translate(
+                                      'savedObjectsManagement.recentWorkSection.updatedAt',
+                                      {
+                                        defaultMessage: 'Updated',
+                                      }
+                                    )}
+                                :{' '}
                               </EuiText>
                             </EuiFlexItem>
                             <EuiFlexItem grow={1} style={{ textAlign: 'right' }}>
```

Comment:
```suggestion
                            <EuiFlexItem grow={1}  className="eui-textRight">
```

Reply:
Thank you for the comments. I 'm sorry I have a later commit that addresses this issue. Please check the latest commit.